keyboards/acheron/athena/readme.md

```diff
@@ -0,0 +1,25 @@
+# Acheron Athena QMK firmware
+
+![athena](https://acheronproject.com/images/athena/IMG_9653.jpg)
```

Comment:
Please use a smaller image here.

Reply:
The link has been updated with a reduced image.

Reply:
Doesn't seem to have been changed.
![image](https://user-images.githubusercontent.com/4781841/179341898-bb7c5568-c0d5-4b87-a4de-9bdfc693d5c5.png)

Reply:
It did, not in pixel size but density. Size now is ten times less.

Reply:
Not really what I meant by smaller...

Reply:
Sorry. I giggled

Reduced it to 20% pixel size

Reply:
Ok I reduced it to 20% pixel size lol